luigi/contrib/sqla.py

```diff
@@ -0,0 +1,215 @@
+# Copyright (c) 2015 Gouthaman Balaraman
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+"""
+This file implements RDBMS target using SQLAlchemy. This opens up
+possibilities to use other database targets as long as it is supported
+by SQLAlchemy. In order to write to a database, one should subclass the
+CopyToTable defined in this file. Please see the sqla_test.py for
+some simple use cases.
+
+Author : Gouthaman Balaraman
+Date : 01/02/2015
+"""
+
+
+import abc
+import logging
+import luigi
+import datetime
+import itertools
+from sqlalchemy import Table, MetaData, Column, String, DateTime, create_engine, select
+
+logger = logging.getLogger('luigi-interface')
+
+
+class SQLAlchemyTarget(luigi.Target):
+    """Target for a resource in database using SQLAlchemy.
+
+    This will rarely have to be directly instantiated by the user"""
+    marker_table = luigi.configuration.get_config().get('sqlalchemy', 'marker-table', 'table_updates')

```

Comment:
I would prefer it not to hit configuration in the global scope. Can you set it `None` and look it up when needed?